### PR TITLE
Add Location(Kawachi)

### PR DESCRIPTION
**Pull request by wagatan**
I added location drone field of Japan.
In japan,we use this field some times.
Thanks.

ドローンフィールド KAWACHI
〒300-1403 茨城県稲敷郡河内町金江津7501-2
http://dronefield-kawachi.irobotics.co.jp/

### CODE REVIEW

**Comment by muramura** [comment]
@wagatan san..
The description is written in English.
That is because there are many members and contributors who do not understand Japanese.
Is the address below?

Location:
7501-2, Kanaetsu
Kawachi-machi,Inashiki-gun,Ibaraki,JAPAN

**Comment by wagatan** [comment]
> @wagatan san..
> The description is written in English.
> That is because there are many members and contributors who do not understand Japanese.
> Is the address below?
> 
> Location:
> 7501-2, Kanaetsu
> Kawachi-machi,Inashiki-gun,Ibaraki,JAPAN

@muramura san
Yes,thanks for your kind support.

**Comment by muramura** [comment]
I think this flight facility is very good for testing various drone.
The flight facilities at this river can be used at school buildings, gymnasiums, river beds and rivers.
At this facility, Ry of Randy's chief Drones Software Engineer School tested a plane, a multicopter, a boat, and navigation.
The river can use 10 km.

File: Tools/autotest/locations.txt [hunk 1]
```diff
@@ -52,3 +52,5 @@ UChicago=41.7886079,-87.59871329999999,191,0
 Yachiyo=35.770481,140.107929,36,90
 Tyndall=30.0142232,-85.4984701,7,0
 Elvenes=68.871422,17.986690,17,256
+Kawachi=35.879129,140.339683,7,0
```
Comment (muramura) [inline Tools/autotest/locations.txt]: The nose is not pointing towards the riverbed field.
I think it would be better to point the nose towards the field.

**Comment by rmackay9** [comment]
Merged with a change to the commit line, txs!

**Pull request closed without merging**